IssueComment {issueCommentUpdatedAt = GithubDate {fromGithubDate = 2012-08-17 14:04:22 UTC}, issueCommentUser = GithubUser {githubOwnerAvatarUrl = "https://secure.gravatar.com/avatar/e782561a98d0e131cbd4980f62853597?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png", githubOwnerLogin = "mikeg-de", githubOwnerUrl = "https://api.github.com/users/mikeg-de", githubOwnerId = 604842, githubOwnerGravatarId = "e782561a98d0e131cbd4980f62853597"}, issueCommentUrl = "https://api.github.com/repos/fail2ban/fail2ban/issues/comments/7818654", issueCommentCreatedAt = GithubDate {fromGithubDate = 2012-08-17 14:04:22 UTC}, issueCommentBody = "Any idea what causes this issue?", issueCommentId = 7818654}